.github/workflows/test-energyflow.yml

```diff
@@ -41,6 +42,7 @@ jobs:
       - name: Install test prerequisites
         run: |
           python -m pip install cython
+          python -m pip install tf-keras
```

Comment:
Having the `cython` and `tf-keras` installs happen in the same `pip` command would be slightly nicer, but nothing "wrong" here.